src/app/models/Harvest.spec.ts

```diff
@@ -1,32 +1,40 @@
 import { Harvest, HarvestStatus } from "@models/Harvest";
 import { generateHarvest } from "@test/fakes/Harvest";
 
-describe("isAbortable", () => {
-  const createModel = (harvestStatus: HarvestStatus) =>
+const createModel = (harvestStatus: HarvestStatus) =>
     new Harvest(generateHarvest({ status: harvestStatus }));
 
-  const transitionalHarvestStates: HarvestStatus[] = [
+describe("isAbortable", () => {
+  const abortableHarvestStates: HarvestStatus[] = [
     "metadataReview",
     "uploading",
     "newHarvest"
   ];
 
-  const notTransitionalHarvestStates: HarvestStatus[] = [
+  const notAbortableHarvestStates: HarvestStatus[] = [
     "scanning",
     "metadataExtraction",
     "processing",
     "complete"
   ];
 
-  transitionalHarvestStates.forEach((harvestStatus: HarvestStatus) => {
+  abortableHarvestStates.forEach((harvestStatus: HarvestStatus) => {
     it(`should return true for a Harvest with the status of ${harvestStatus}`, () => {
       expect(createModel(harvestStatus).isAbortable()).toBeTrue();
     });
   });
 
-  notTransitionalHarvestStates.forEach((harvestStatus: HarvestStatus) => {
+  notAbortableHarvestStates.forEach((harvestStatus: HarvestStatus) => {
     it(`should return false for a Harvest with the status of ${harvestStatus}`, () => {
       expect(createModel(harvestStatus).isAbortable()).toBeFalse();
     });
   });
 });
+
+describe("canUpdate", () => {
+  // as canUpdate is called by isAbortable
+  // all other Harvest statuses are covered by the isAbortable test
+  it("should return true for a Harvest with the status of complete", () => {
+    expect(createModel("complete").canUpdate).toBeTrue();
```

Comment:
i'd encourage you to test all the states. 

the shortcut method is ok in the implementation, but the test of implementation should be independent